catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/operation/impl/UpdateRequestImpl.java

```diff
@@ -92,6 +92,23 @@ public UpdateRequestImpl(String[] ids, List<Metacard> metacards)
     }
 
     /**
+     * Instantiates a new UpdateRequestImpl from a set of destinations,
+     * an array of ids, and a list of {@link Metacard}
+     *
+     * @param destinations the destination ids this request should be sent to
+     * @param ids          the ids associated with the {@link Metacard} list
```

Comment:
I think the Catalog Framework should protect against that if we think that might happen.  `validateUpdateRequest` could verify that the attribute values match the update attribute key values.

https://github.com/codice/ddf/blob/08651d5cc59e559b940b4faaec6334657948c18c/catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/UpdateOperations.java#L349